README.md

```diff
@@ -216,6 +216,41 @@ The benefits of using the plain version of SDK are:
 - The ability to scope CMA client instance to a specific `spaceId`, `environmentId`, and `organizationId` when initializing the client.
   - You can pass a concrete values to `defaults` and omit specifying these params in actual CMA methods calls.
 
+### App Framework
+
+Starting [`@contentful/app-sdk@4`](https://github.com/contentful/app-sdk) you can use this client to issue requests
+from your [Contentful app](https://www.contentful.com/developers/docs/extensibility/app-framework/).
+
+A dedicated [Adapter](https://github.com/contentful/contentful-management.js/blob/2350b47053459694b21b19c71025632fe57815cc/lib/common-types.ts#L493-L495)
+grants your app access to the supported space-environment scoped entities without compromising on security as you won't
+need to expose a management token, and without coding any additional backend middleware.
+
+```javascript
+const contentfulApp = require('@contentfu/app-sdk')
+const contentful = require('contentful-management')
+
+contentfulApp.init((sdk) => {
+  const cma = contentful.createClient(
+    { apiAdapter: sdk.cmaAdapter },
+    {
+      type: 'plain',
+      defaults: {
+        environmentId: sdk.ids.environment,
+        spaceId: sdk.ids.space,
+      },
+    }
+  )
+
+  // ...rest of initialization code
+})
+```
+
+> **Please Note**
+>
+> Requests issued by means of App SDK's adapter will count towards the same rate limiting quota as the ones issued by
+> other APIs (e.g., Space API). Ultimately, they will all fall into the same bucket as the calls performed by the
```

Comment:
I am not sure what you mean with "other APIs". APIs in within the App SDK?

Reply:
```suggestion
> other APIs (e.g. Space API). Ultimately, they will all fall into the same bucket as the calls performed by the
```

Reply:
> I am not sure what you mean with "other APIs". APIs in within the App SDK?

Yeah, rephrasing it to make it clearer